source/server/config/access_log/grpc_access_log.cc

```diff
@@ -0,0 +1,73 @@
+#include "server/config/access_log/grpc_access_log.h"
+
+#include "envoy/registry/registry.h"
+#include "envoy/server/filter_config.h"
+
+#include "common/access_log/grpc_access_log_impl.h"
+#include "common/common/macros.h"
+#include "common/config/well_known_names.h"
+#include "common/grpc/async_client_impl.h"
+#include "common/protobuf/protobuf.h"
+
+#include "api/filter/accesslog/accesslog.pb.validate.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+class GrpcAccessLogClientFactoryImpl : public AccessLog::GrpcAccessLogClientFactory {
+public:
+  GrpcAccessLogClientFactoryImpl(Upstream::ClusterManager& cluster_manager,
```

Comment:
Where is the access log server cluster referred by this  cluster_name configured? i could not find such config in boostrap config. Is it done differently here?

Reply:
ok. But how does envoy know that it should connect to that cluster? is it by some standard name? If you LDS, Ratelimiting etc, they have specific configuration in bootstrap.yaml that tells what cluster should be used for them. or Am I missing some thing here?

Reply:
It's not a filter, but an access log provider which has a config. Take a look at the integration test. That has a complete config.